databases/week1/hyf_lesson1_Homework.sql

```diff
@@ -0,0 +1,81 @@
+/* Find out how many tasks are in the task table */
+select
+  COUNT(id)
```

Comment:
Instead of making a newline for each identifier, you should instead write
`select COUNT(id)
fromtask
.....`

That makes it much easier to read

Reply:
thanks.. but its because of extension i have its happening